class1.md

```diff
@@ -0,0 +1,261 @@
+
+By the end of this course you should be able to:   
+
+* Identify how machine learning can be applied in biology research and form a problem statement.
+* Apply the concepts of exploratory data analysis, experimental design, and ethics machine learning.
+* Recognize the four main paradigms of machine learning as supervised, unsupervised, semi-supervised, and reinforcement learning.
+* Differentiate between common approaches and methods for supervised and unsupervised machine learning.
+* Recognize and have a basic understanding of common approaches to supervised (regression, classification) and unsupervised (clustering, dimensionality reduction) machine learning.
+
+Today's lecture will focus on providing a broad introduction and structure to many of the concepts we will cover throughout the rest of this course.
+
+By the end of this class you should be able to:
+
+* Define and understand the limitations of machine learning.
+* Understand basic terminology like algorithm, inputs, outputs, supervised, unsupervised.
+* Differentiate between unsupervised and supervised machine learning methods.
+
+### What **isn't** machine learning?
+
+Buzzwords like ‘machine learning’, ‘deep learning’, and ‘artificial intelligence’ get thrown around quite a bit these days. Sometimes they seemed to be used as synonyms, other times as distinct entities. Because of this, it can be hard to determine exactly what these terms mean to the layperson. So let’s start with what machine learning *isn't*.
+
+Like any new technique that gains traction and buzz in the field, suddenly everyone wants to jump on the machine learning bandwagon. However, machine learning isn't a shiny, new toy to be showed off or assessory to be casually added to an analysis. As with all statistical methods, it can be powerful when applied correctly or extremely misleading and outright wrong when applied incorrectly.
+
+<p align="center">
+  <img src="images/dashml.jpeg" width="250" />
+  <img src="images/mlPile.png" width="250" /> 
+  <img src="images/simpleProblem.png" width="250" />
+</p>
+
+**Machine learning is NOT:**
+
+* Magic
+* A secret sauce
+* Seasoning to be sprinkled liberally over a mundane application
+* A substitute for a good understanding of your problem
+
+### What **is** machine learning?
+
+Recent advances in compute power, cloud technologies, data collection and data generation have thrust machine learning into the mainstream. Increasingly machine learning is looked to as a tool to solve Big Complex Problems like cancer. Often though the term machine learning is used as a sort of black box, but what actually is machine learning?
+
+Let’s orient ourselves to the terms I mentioned in the previous slides that you may be familiar with (ML, AI, Deep learning) to get a sense of how things fit together.
+
+<p align="center">
+  <img align="middle" width="500" src="images/aiMlDl.png">
+</p>
+
+* AI is a term that embodies anything that enables computers to act more like humans.
```

Comment:
It's confusing that this is a nested list, since machine learning is actually a part of AI, etc. Maybe keep the three main points (AI, ML, deep learning) and turn the bullet points below them into sentences. For example:

* AI is a term that embodies anything that enables computers to act more like humans.
* Machine learning is a field of study within AI, similar to how molecular biology is a specific focus in biology. The focus of this field is on the extraction of patterns from large datasets.
* Deep learning is an example of a collection of similar machine learning methods.
...